pyproject.toml

```diff
@@ -56,6 +56,7 @@ sphinx-copybutton = "^0.5.2"
 ruff = "^0.0.292"
 nbmake = "^1.4.3"
 nbsphinx = "^0.9.3"
+omfit_classes = "^3.2024.09"
```

Comment:
Yeah we will look into https://github.com/freegs-plasma/FreeQDSK, it looks cool! I hadn't heard of this. 

I think it would be nice to have this option inside of `cfspopcon` - many modelers will appreciate the option to work directly from a gEQDSK. And further this will open up an option to improve some calculations, such as volume integrals, with gEQDSKs at other points within the code. 

I guess an alternative would be to develop a suite of supporting scripts that would be distinct from `cfspopcon` but very much married with the code? This could be pointing already to MOSAIC...

Reply:
> I guess an alternative would be to develop a suite of supporting scripts that would be distinct from cfspopcon but very much married with the code? This could be pointing already to MOSAIC...

Yes. I think this would be a good idea to solve the problem.